probe/load_data.py

```diff
@@ -121,8 +121,11 @@ def get_metadata(self):
             'run_name': self.run_name
         }
 
-    def save_computed_embeddings(self, sentences, inputs, indices, pools, metadata):
-        folder = os.path.join('cache', self.run_name)
+    def save_computed_embeddings(self, sentences, inputs, indices, pools, metadata, save_sub_location=None):
```

Comment:
this is so that generating the individual sentence embeddings won't override the other embeddings and so that they can be cached to be used again in subsequent runs

Reply:
let me know if there's a different/better way to handle that though